examples/query-actions/query_actions.md

```diff
@@ -0,0 +1,11 @@
+# Query Actions
+
+The clauses in a query-action are executed in the same way as the clauses in a query-expr.
+
+In the same way the query-pipeline is built in query expressions, the query pipeline can be built using other clauses such as `let` clause, `join` clause, `order by` clause, `where clause` and `limit` clause except the `select` clause.
```

Comment:
Our query expression-related BBEs don't mention about the `query pipeline`? Wouldn't it be confusing for a new user, to what a query pipeline is since we have not explained it? Maybe we can word this differently.